documentation/modules/overview/con-overview-components-cluster-operator.adoc

```diff
@@ -27,6 +27,11 @@ The Cluster Operator can also deploy (through configuration of the `Kafka` resou
 
 The Topic Operator and User Operator function within the Entity Operator on deployment.
 
+You can use the Cluster Operator with a deployment of Strimzi Drain Cleaner to help with pod evictions.
+By deploying the Strimzi Drain Cleaner, you can use the Cluster Operator to move Kafka pods instead of Kubernetes.
+Strimzi Drain Cleaner annotates pods being evicted with a rolling update annotation.
+The annotation informs the Cluster Operator to perform the rolling update.
```

Comment:
Can we add some link to more details about the Drain Cleaner? I think this is correct, but it should have a link to click through for more details about the pod disruption budgets etc.